KEYHOLDER.md

```diff
@@ -30,6 +35,17 @@ You will be prompted to reset your hardware key and set a PIN. Choose a PIN betw
 
 This will output three files (a public key, device certificate, and hardware certificate) in a directory named with your serial number `ceremony/YYYY-MM-DD/keys/${SERIAL_NUM}`.
 
+Troubleshooting: If you hit the error
+```
+error: connecting to pscs: the Smart card resource manager is not running
+```
+
+then run the following to start the pcsc daemon:
```

Comment:
@haydentherapper did you have to install anything before running this step?

Reply:
Not on mac. On linux, `libpcsclite-dev`.

Reply:
Awesome, thanks! That's already included